frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts

```diff
@@ -123,7 +123,7 @@ export class QueryGetIcalUrlModalComponent extends OpModalComponent implements O
       return;
     }
 
-    let icalUrl = '';
+    const icalUrl = '';
```

Comment:
This seems to be an unused constant. We can remove it and appease eslint :)
```suggestion
```

Reply:
I just did not do anything else to the file. Will remove

Reply:
It was fine either way. I know eslint starts complaining the moment you touch a file. Left the commit suggestion there just so you didn't have to go back and do it :)